morpheus/llm/services/openai_chat_service.py

```diff
@@ -231,18 +264,37 @@ def generate_batch(self, inputs: dict[str, list[str]]) -> list[str]:
         results = []
         for (i, prompt) in enumerate(prompts):
             assistant = assistants[i] if assistants is not None else None
-            results.append(self._generate(prompt, assistant))
+            if (return_exceptions):
+                results.append(self._generate(prompt, assistant, return_exceptions=True))
+            else:
+                results.append(self._generate(prompt, assistant, return_exceptions=False))
```

Comment:
Is the if statement needed here? Couldn't it just be:
```
results.append(self._generate(prompt, assistant, return_exceptions=return_exceptions))
```

Reply:
This allows the type hinting to actually work and prevents pylint errors all over the place